crossing-training.lic

```diff
@@ -231,13 +231,21 @@ class CrossingTraining
     return unless @settings.tithe
     delta = Time.now - (UserVars.tithe_timer || Time.now - 14_400)
     return if (delta < 4 * 60 * 60 && DRStats.paladin?) || (delta < 60 * 10 && DRStats.cleric?)
+    currency = get_data('town')[@settings.hometown]['currency']
```

Comment:
PR #1912 is adding a helper that does this for you, let's keep an eye on which PR gets merged first.